io/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "bitcoin-io"
+version = "0.1.0"
+authors = ["Matt Corallo <birchneutea@mattcorallo.com>"]
+license = "CC0-1.0"
+repository = "https://github.com/rust-bitcoin/rust-bitcoin"
+documentation = "https://docs.rs/bitcoin-io/"
+description = "Simple I/O traits for no-std (and std) environments"
+categories = ["no-std"]
+keywords = [ "io", "no-std" ]
+readme = "README.md"
+edition = "2018"
+exclude = ["tests", "contrib"]
+
+[features]
+default = ["std"]
+std = []
+alloc = ["core2/alloc"]
```

Comment:
We're using local dependencies in this repo so you should be able to drop it from hashes in the same PR (we will release properly anyway). Unless I'm missing something.

Reply:
I don't think it's necessary. Neither `bitcoin` nor `hashes` directly depend on `hex-conservative/core2` so you can probably just remove those and it should be possible to build it.

Reply:
If we need a bridge I'd just provide a few wrapper structs for `core2` that simply delegate to the inner field. Also I suspect it may be better to do the same for `std` but maybe less so.

Reply:
Not a blocker for me, it'd just make it easier to review.

Reply:
>  Neither bitcoin nor hashes directly depend on hex-conservative/core2 so you can probably just remove the feature dependency and it should be possible to build it.

That is exactly why I did https://github.com/rust-bitcoin/rust-bitcoin/pull/2123